### DISCUSSION TITLE
Send notification when episode moves to backlog

### DISCUSSION THREAD

**Discussion by jandreaucodes** [pymedusa/Medusa#8279] [Ideas & Feature requests]
My wife wants some shows that aren't always torrented. So I'm looking to use pyMedusa (for those shows) more as a calendar to notify me when a new episode is available. So, in the event an episode doesn't make it to a torrent, we can go to that network's app and watch it there.

It'd be awesome if I could get a notification when an episode falls off my "schedule" and moves to the backlog. Then I'd know to go look on the network's app.

I don't see any options for any notification types other than when episodes are snatched or downloaed. But I'm wondering if anyone has a suggestion that'll work, other than just checking the "Schedule" page frequently (which I do more than enough, anyway).

**Comment by p0psicles** [comment]
So you'd like a notification when an ep is aired? Or when it's aired but medusa can't find a result after x days?

**Comment by jandreaucodes** [comment]
I think after X days would work. 

Similar to how an episode falls off the Schedule after X days (If memory serves, there’s a setting for how many days that is.)

So, yeah, episode airs, X days passes, Notification gets sent.

**Comment by p0psicles** [comment]
This is quite a niche request. But soon I need to create a new api for the schedule page. I could make sure you can request shows scheduling, and optionally pass paramaters for airdate offsets. Then with a small script you could poll it, and trigger notifications. 

Alternative we could at some point extend the notification api. But that's not the highest of prio. First I need to implement the new frondend.
